src/core/xref.js

```diff
@@ -72,6 +84,71 @@ export async function run(conf, elems) {
   addDataCiteToTerms(results, xrefMap, conf);
 }
 
+/**
+ * converts conf.xref to object with url and spec properties
+ */
+function normalizeConfig(xref) {
+  const xrefDefaults = {
+    url: API_URL,
+    specs: null,
+  };
+
+  const normalizedConfig = {};
+  const type = Array.isArray(xref) ? "array" : typeof xref;
+  switch (type) {
+    case "boolean":
+      // ... true
+      Object.assign(normalizedConfig, xrefDefaults);
+      break;
+    case "string":
+      if (profiles[xref]) {
+        // We have the profile
+        Object.assign(normalizedConfig, xrefDefaults, {
+          specs: profiles[xref],
```

Comment:
Users don't care for case :) 

```suggestion
          specs: profiles[xref.toLowerCase()],
```